### PR TITLE
Stretch

### PR DESCRIPTION
**Pull request by b4yuan**
stretch psdr... dynamically change contacts from user input, make an emergency buttons how up when bac reaches a certain level

### CODE REVIEW

**Review by jtrowe08 — DISMISSED** [review]

File: src/pages/Emergency.js [hunk 1]
```diff
@@ -80,7 +79,7 @@ export default function Emergency({ navigation }) {
         <Text style={styles.emergencyButtonText}>Call Emergency Services</Text>
       </TouchableOpacity>
 
-      {contacts.map((contact, index) => (
+      {emergencyContacts.map((contact, index) => (
```
Comment (khauersp) [inline src/pages/Emergency.js]: appreciate the descriptive variable name

File: src/pages/Home.js [hunk 2]
```diff
@@ -125,29 +214,34 @@ export default function Home({ navigation }) {
 
     if (bac > 10 && bac <= 20) {
       newRiskMessage = 'Medium risk';
-    } else if (bac > 20) {
+    } else if (bac > -5) {
```
Comment (khauersp) [inline src/pages/Home.js]: Why -5?

File: src/pages/Settings.js [hunk 3]
```diff
@@ -52,14 +52,22 @@ export default function SettingsScreen({ navigation }) {
...
-          let ec5n = emergencyContact1Name
-          let ec5p = emergencyContact1Phone
+          let ec2n = emergencyContact2Name
+          let ec2p = emergencyContact2Phone
+          let ec3n = emergencyContact3Name
+          let ec3p = emergencyContact3Phone
+          let ec4n = emergencyContact4Name
+          let ec4p = emergencyContact4Phone
+          let ec5n = emergencyContact5Name
+          let ec5p = emergencyContact5Phone
```
Comment (khauersp) [inline src/pages/Settings.js:62]: So I know absolutely nothing about js but why this and not just put the variables straight into the arrays?

**Review by khauersp — DISMISSED** [review]

File: src/pages/Home.js (hunk 2 above)
Comment (b4yuan) [reply src/pages/Home.js]: arbitrary vals for testing

File: src/pages/Settings.js (hunk 3 above)
Comment (b4yuan) [reply src/pages/Settings.js:62]: I was having some issues with that and this was just the dumb solution

File: src/pages/Home.js (hunk 2 above)
Comment (khauersp) [reply src/pages/Home.js]: Ah okay cool

File: src/pages/Settings.js (hunk 3 above)
Comment (khauersp) [reply src/pages/Settings.js:62]: Ah nice, as long as it works it doesn't have to look pretty

**Review by GraysonNocera — APPROVED** [review]
after my in-depth pull request review, I have deemed this to be wonderful work.

**Review by khauersp — APPROVED** [review]
looks great

**Pull request merged**